components/automate-ui/src/app/entities/layout/layout.facade.ts

```diff
@@ -70,39 +79,7 @@ export class LayoutFacadeService {
     this.contentHeight = `calc(100% - ${this.headerHeight})`;
   }
 
-  showDashboardsSidebar(): void {
-    this.store.dispatch(new UpdateSidebarMenuGroups(
-      this.layoutSidebarService.getDashboardsSidebar()
-    ));
-  }
-
-  showApplicationsSidebar(): void {
-    this.store.dispatch(new UpdateSidebarMenuGroups(
-      this.layoutSidebarService.getApplicationsSidebar()
-    ));
-  }
-
-  showInfastructureSidebar(): void {
-    this.store.dispatch(new UpdateSidebarMenuGroups(
-      this.layoutSidebarService.getInfastructureSidebar()
-    ));
-  }
-
-  showComplianceSidebar(): void {
-    this.store.dispatch(new UpdateSidebarMenuGroups(
-      this.layoutSidebarService.getComplianceSidebar()
-    ));
-  }
-
-  showSettingsSidebar(): void {
-    this.store.dispatch(new UpdateSidebarMenuGroups(
-      this.layoutSidebarService.getSettingsSidebar()
-    ));
-  }
-
-  showUserProfileSidebar(): void {
-    this.store.dispatch(new UpdateSidebarMenuGroups(
-      this.layoutSidebarService.getUserProfileSidebar()
-    ));
+  showSidebar(sidebarName: string) {
```

Comment:
what's the difference between enabling a sidebar and showing a sidebar?

Reply:
@blakestier true, we can rename it. We call this in the pages to specify which sidebar to display or enable. this.layoutFacade.showSidebar(Sidebar.Settings);